synthtool/gcp/templates/java_library/.kokoro/continuous/propose_release.cfg

```diff
@@ -0,0 +1,53 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Build logs will be here
+action {
+  define_artifacts {
+    regex: "**/*sponge_log.xml"
+  }
+}
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Use the trampoline script to run in docker.
+build_file: "{{ metadata['repository_name'] }}/.kokoro/trampoline.sh"
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/node:10-user"
```

Comment:
Is this supposed to be the node image?

Reply:
Yeah, release-please is nodejs-based so we explicitly allow nodejs for this job